ipn/ipnlocal/c2n.go

```diff
@@ -83,6 +83,8 @@ func (b *LocalBackend) handleC2N(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 		writeJSON(res)
+	case "/sockstats":
+		b.sockstatLogger.WriteLogs(w)
```

Comment:
Should probably do a `w.Header().Set("Content-Type", "text/plain")` to ensure the browser doesn't try to interpret the output as HTML.

Reply:
done